src/components/ProductTabs.jsx

```
@@ -0,0 +1,119 @@
+        </Tabs.Trigger>
+      </Tabs.List>
+      
+      <Tabs.Content 
+        value="description" 
+        className="py-6 text-[var(--md-sys-color-on-surface)]"
+      >
+        {product.description ? (
+          <div className="prose prose-lg max-w-none">
+            {/* If description is HTML content, use dangerouslySetInnerHTML */}
+            {product.description.includes('<') ? (
+              <div dangerouslySetInnerHTML={{ __html: product.description }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)